f5/bigip/resource.py

```diff
@@ -295,6 +305,37 @@ def _handle_requests_params(self, kwargs):
             requests_params.update({'params': params})
         return requests_params
 
+    def _check_exclusive_parameters(self, **kwargs):
```

Comment:
The idea is that we have 100% code coverage for every new piece of code that codes in. If you haven't done with with pytest, you can install pytest-cov and run it like so: `py.test --cov-report term --cov=resource.py test/test_resource.py`